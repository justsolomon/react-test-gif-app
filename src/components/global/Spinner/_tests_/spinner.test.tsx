import { render } from "@testing-library/react";
import Spinner from "../";

describe("<Spinner />", () => {
  it("should render the Spinner component properly", () => {
    const { asFragment } = render(<Spinner />);

    expect(asFragment()).toMatchSnapshot();
  });
});
