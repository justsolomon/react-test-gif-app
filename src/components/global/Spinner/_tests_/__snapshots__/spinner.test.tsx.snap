// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render the Spinner component properly 1`] = `
<DocumentFragment>
  <svg
    height="36px"
    preserveAspectRatio="xMidYMid"
    style="margin: auto; background: none; display: block; shape-rendering: auto;"
    viewBox="0 0 100 100"
    width="36px"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <circle
      cx="50"
      cy="50"
      fill="none"
      r="40"
      stroke="#3182ce"
      stroke-dasharray="188.49555921538757 64.83185307179586"
      stroke-width="10"
    >
      <animatetransform
        attributeName="transform"
        dur="0.7407407407407407s"
        keyTimes="0;1"
        repeatCount="indefinite"
        type="rotate"
        values="0 50 50;360 50 50"
      />
    </circle>
  </svg>
</DocumentFragment>
`;
